Makefile

```diff
@@ -467,13 +467,17 @@ check-generated:
 		((git diff $$(find . -name '*.pb.desc') | cat) && \
 		(echo "Please run 'make generate' when making changes to proto files and check-in the generated file changes" && false))
 
+LICENSES = Apache-2.0,BSD-2-Clause,BSD-2-Clause-FreeBSD,BSD-3-Clause,MIT,ISC,Python-2.0,PostgreSQL,X11,Zlib
```

Comment:
I don't like that we list allowed licenses in 2 places. The other place is:https://github.com/lima-vm/lima/blob/56702e0e525207e808e6d79bcf7296b6c400f050/.github/workflows/test.yml#L68)

Can't we just put that line into a textfile and use it like this:

```shell
go-licenses check … --allowed-licenses="$(cat allowed-licenses.txt)"
```

Reply:
Probably GHA should just run `make lint` ?

Reply:
The only thing is how to install the tools